test/plugin/scenarios/spring-kafka-2.2.x-scenario/support-version.list

```diff
@@ -14,16 +14,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+2.2.14.RELEASE
 2.2.13.RELEASE
+2.2.12.RELEASE
 2.2.11.RELEASE
+2.2.10.RELEASE
 2.2.9.RELEASE
-2.2.7.RELEASE
-2.2.5.RELEASE
-2.2.3.RELEASE
-2.1.14.RELEASE
```

Comment:
Aren't supporting 2.1.x? I think you should create a new test case?

Reply:
I have seen that version 2.1.x, does not have this method, and i change `Supported-list.md` to 2.2.x